ddtrace/contrib/django/utils.py

```diff
@@ -57,3 +58,36 @@ def set_tag_array(span, prefix, value):
     else:
         for i, v in enumerate(value, start=0):
             span.set_tag("{0}.{1}".format(prefix, i), v)
+
+
+def get_request_uri(request):
```

Comment:
if we can add some tests for this, then 👍

Reply:
I've added a test for the issue fixed here:

https://github.com/DataDog/dd-trace-py/pull/1235/files#diff-add6aa0921e61690adb54ee50c875dd3R121

I am reluctant to add unit tests for this utility function as there are already tests that check the value of `http.url` which makes use of `get_request_uri`.